hackedserver-bungeecord/src/main/java/org/hackedserver/bungee/listeners/CustomPayloadListener.java

```diff
@@ -35,11 +35,12 @@ public void onPluginMessageReceived(PluginMessageEvent event) {
     }
 
     private void performActions(Action action, ProxiedPlayer player, TagResolver.Single... templates) {
+        if (player.hasPermission("hackedserver.bypass")) return;
```

Comment:
If you want the same behaviour about version 2.X
The bypass show alert,  it bypass only the action